Modelica/Mechanics/MultiBody/Examples/Systems/RobotR3.mo

```diff
@@ -46,8 +46,7 @@ package RobotR3
     connect(axis.flange, load.flange_a)
```

Comment:
Also change starteAngle / endAngle in lines 14/15 to SI.Conversions.NonSIunits.Angle_deg?